modules/seqexec/web/client/src/main/scala/seqexec/web/client/actions.scala

```diff
@@ -56,14 +56,16 @@ object actions {
       extends Action
 
   // Actions related to executing sequences
-  final case class RequestRun(s:         Observation.Id) extends Action
-  final case class RequestSync(s:        Observation.Id) extends Action
-  final case class RequestPause(s:       Observation.Id) extends Action
-  final case class RequestCancelPause(s: Observation.Id) extends Action
-  final case class RequestStop(id:       Observation.Id, step: Int) extends Action
-  final case class RequestAbort(id:      Observation.Id, step: Int) extends Action
-  final case class RequestObsPause(id:   Observation.Id, step: Int) extends Action
-  final case class RequestObsResume(id:  Observation.Id, step: Int) extends Action
+  final case class RequestRun(s:               Observation.Id) extends Action
+  final case class RequestSync(s:              Observation.Id) extends Action
+  final case class RequestPause(s:             Observation.Id) extends Action
+  final case class RequestCancelPause(s:       Observation.Id) extends Action
+  final case class RequestStop(id:             Observation.Id, step: Int) extends Action
```

Comment:
We have a StepId alias for the `Int` We may eventually replace it by some richer type

Reply:
Fixed